drivers/ata8510/include/ata8510_internals.h

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (C) 2016 Antonio Galea <antonio.galea@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     drivers_ata8510
+ * @{
+ *
+ * @file
+ * @brief       Internal interfaces for ATA8510 driver
+ *
+ * @author      Antonio Galea <antonio.galea@gmail.com>
+ * @author      Roberto Asquini <bobasquins@gmail.com>
+ */
+
+#ifndef ATA8510_INTERNALS_H
+#define ATA8510_INTERNALS_H
+
+#include <stdint.h>
+
+#include "ata8510.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define ATA8510_MAX_PACKET_LEN  (255)
```

Comment:
Is this entirely software defined or is there a hardware limitation?

Reply:
Purely software: I've used uint8_t as the buffer lenght type. Could be changed if needed.